x=0
try:
    print(x)
except NameError:
    print("x has not called so far")